tools/config/__init__.py

```diff
@@ -540,7 +556,7 @@ def regions(self):
             raise ConfigException(
                 "Bootloader build requested but no bootlader configuration")
 
-    def _generate_booloader_build(self, target_overrides, rom_start, rom_size):
+    def _generate_bootloader_build(self, target_overrides, rom_start, rom_size, sectors):
```

Comment:
👍 For the typo correction. Do we have to pass sectors in? Could this function also do `self.sectors`?

Reply:
Good call.